runsc/boot/vfs.go

```diff
@@ -585,6 +585,12 @@ func (c *containerMounter) configureOverlay(ctx context.Context, conf *config.Co
 	// filesystem specific options.
 	upperOpts := *lowerOpts
 	upperOpts.GetFilesystemOptions = vfs.GetFilesystemOptions{InternalMount: true}
+	if mountConf.UpperSize != "" {
+		if upperOpts.GetFilesystemOptions.Data != "" {
+			upperOpts.GetFilesystemOptions.Data += ","
+		}
```

Comment:
This is guaranteed to be false right? Since we just initialized `upperOpts.GetFilesystemOptions` on line 587 above.

Reply:
Yes. I added here so that next time someone will add another option here or above, they won't forget to add a comma. I can revert.